apps/web/components/sections/project/success-gallery.tsx

```
@@ -0,0 +1,36 @@
+
+interface SuccessGalleryProps {
+  items: { src: string; alt: string }[];
+  onMediaClick?: (index: number) => void; 
+}
+
+const SuccessGallery: React.FC<SuccessGalleryProps> = ({ items, onMediaClick }) => {
+  return (
+    <div className="bg-white rounded-lg p-6 shadow-md border border-gray-200">
+      <div className="flex items-center justify-between mb-4">
+        <h3 className="text-lg font-bold mb-4">Success Gallery</h3>
+        <button className="text-sm font-medium text-blue-600 hover:underline">View All</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)